src/sentry/static/sentry/app/components/events/interfaces/richHttpContent/richHttpContentClippedBoxBodySection.tsx

```diff
@@ -32,7 +32,7 @@ const RichHttpContentClippedBoxBodySection = ({
     if (meta && typeof value === 'string') {
       return (
         <AnnotatedText
-          value={value}
+          value={<pre>{value}</pre>}
```

Comment:
Can you wrap the `AnnotatedText` in `pre`? This will break for certain tooltips (untested, yolo)

Reply:
yes sure ;-)